stdlib/public/core/CTypes.swift

```diff
@@ -116,6 +116,8 @@ public typealias CLongDouble = Double
 #elseif os(FreeBSD)
 #if arch(x86_64) || arch(i386)
 public typealias CLongDouble = Float80
+#elseif arch(arm64)
+public typealias CLongDouble = Double
```

Comment:
Isn't it Float128 (also for OpenBSD)? I note that there are various comments above about Float128 not being supported, so perhaps this should be omitted for arm64 on *BSD?

Reply:
Thanks @jrtc27! Also cc @3405691582 for OpenBSD